packages/nodes-base/nodes/N8nApi/CredentialDescription.ts

```diff
@@ -0,0 +1,160 @@
+import { INodeProperties } from 'n8n-workflow';
+import { parseAndSetBodyJson } from './GenericFunctions';
+
+export const credentialOperations: INodeProperties[] = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		noDataExpression: true,
+		default: 'create',
+		displayOptions: {
+			show: {
+				resource: ['credential'],
+			},
+		},
+		options: [
+			{
+				name: 'Create',
+				value: 'create',
+				action: 'Create a credential',
+				routing: {
+					request: {
+						method: 'POST',
+						url: '=/credentials',
```

Comment:
If there is no dynamic part `{{ ... }}` in the value the equal sign, at the beginning, can be removed. Else it will execute it as an expression. Meaning it will not break anything, it will just unnecessarily run much more complicated code and so waste CPU cycles.

Reply:
Cleaned up a bit, thanks for the heads-up!